src/libraries/grommet/components.js

```diff
@@ -2063,6 +2064,7 @@ export const components = {
     },
     properties: {
       a11yTitle: '',
+      allowSelectAll: [true, false],
```

Comment:
One question -- in some places it looks like when it's a boolean we just have `true` or `false` as opposed to an array with both. Do you know the difference?

Reply:
I think if you just do `false` or `true` it will present it as a checkbox as opposed to select dropdown which might be nice.

Reply:
Gotcha, I'll switch this to `true`

Reply:
I think that we will need to have a defaultProp of `allowSelectAll` set to true as well. Feels like by default it should be checked?

Reply:
Good note, added `allowSelectAll: true` to the default props